From 998c0160ed78423ca53355d9ef29b73fe266e560 Mon Sep 17 00:00:00 2001
From: "kernelci.org bot" <bot@kernelci.org>
Date: Fri, 3 Jan 2020 16:09:40 +0000
Subject: [PATCH 2/7] STAGING ensure no public bisection email reports are sent

---
 scripts/kci-bisect-push-results | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/scripts/kci-bisect-push-results b/scripts/kci-bisect-push-results
index 8d2fe39ad..fe2bdc6aa 100755
--- a/scripts/kci-bisect-push-results
+++ b/scripts/kci-bisect-push-results
@@ -225,6 +225,8 @@ def send_report(args, log_file_name, token, api):
         cc = set()
         for r in to:
             print("  {}".format(r))
+    else:
+        raise Exception("NOT SENDING PUBLIC EMAIL REPORTS ON STAGING")
 
     data.update({
         'report_type': 'bisect',
-- 
2.39.2

