From 5e8f05ce7d5ec5b960be7889ee337ec888460baa Mon Sep 17 00:00:00 2001
From: "kernelci.org bot" <bot@kernelci.org>
Date: Fri, 3 Jan 2020 16:09:40 +0000
Subject: [PATCH 3/5] STAGING ensure no public bisection email reports are sent

---
 scripts/push-bisection-results.py | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/scripts/push-bisection-results.py b/scripts/push-bisection-results.py
index 8d2fe39ad1a8..fe2bdc6aa3f3 100755
--- a/scripts/push-bisection-results.py
+++ b/scripts/push-bisection-results.py
@@ -225,6 +225,8 @@ def send_report(args, log_file_name, token, api):
         cc = set()
         for r in to:
             print("  {}".format(r))
+    else:
+        raise Exception("NOT SENDING PUBLIC EMAIL REPORTS ON STAGING")
 
     data.update({
         'report_type': 'bisect',
-- 
2.20.1

