From e4b4c36cc09cd3407e86fc8644e3c7a303c5e1d2 Mon Sep 17 00:00:00 2001
From: "kernelci.org bot" <bot@kernelci.org>
Date: Fri, 1 Sep 2023 18:46:13 +0000
Subject: [PATCH 6/6] STAGING use staging JWT in kubernetes template

---
 config/runtime/base/kubernetes.jinja2 | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/config/runtime/base/kubernetes.jinja2 b/config/runtime/base/kubernetes.jinja2
index e17917a3..2b96b072 100644
--- a/config/runtime/base/kubernetes.jinja2
+++ b/config/runtime/base/kubernetes.jinja2
@@ -55,7 +55,7 @@ spec:
           valueFrom:
             secretKeyRef:
               # FIXME: convert to template parameter
-              name: {{ "kci-api-jwt-early-access" }}
+              name: {{ "kci-api-jwt-staging" }}
               key: token
 
         - name: KCI_STORAGE_CREDENTIALS
-- 
2.39.2

