From d217f2adb8913353f1fd7c66e68da862f02e9f6c Mon Sep 17 00:00:00 2001
From: "kernelci.org bot" <bot@kernelci.org>
Date: Fri, 1 Sep 2023 18:46:13 +0000
Subject: [PATCH 6/7] STAGING use staging JWT in kubernetes template

---
 config/runtime/base/kubernetes.jinja2 | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/config/runtime/base/kubernetes.jinja2 b/config/runtime/base/kubernetes.jinja2
index 90a174470..9d80cacc5 100644
--- a/config/runtime/base/kubernetes.jinja2
+++ b/config/runtime/base/kubernetes.jinja2
@@ -69,7 +69,7 @@ spec:
           valueFrom:
             secretKeyRef:
               # FIXME: convert to template parameter
-              name: {{ "kci-api-jwt-early-access" }}
+              name: {{ "kci-api-jwt-staging" }}
               key: token
 
         - name: KCI_STORAGE_CREDENTIALS
-- 
2.39.2

