From cfdaca18efd2d9abbe316b207a89be1b7f07e5e3 Mon Sep 17 00:00:00 2001
From: "kernelci.org bot" <bot@kernelci.org>
Date: Fri, 1 Sep 2023 21:37:43 +0200
Subject: [PATCH 7/7] STAGING use [KernelCI staging] in LAVA jobs

---
 config/runtime/base/lava.jinja2 | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/config/runtime/base/lava.jinja2 b/config/runtime/base/lava.jinja2
index 55083f3a8..1c464926c 100644
--- a/config/runtime/base/lava.jinja2
+++ b/config/runtime/base/lava.jinja2
@@ -17,7 +17,7 @@
 # * rootfs URL
 # * Docker image for QEMU
 
-job_name: "[KernelCI AKS] {{ node.id }} {{ node.name }} {{ node.revision.describe }}"
+job_name: "[KernelCI staging] {{ node.id }} {{ node.name }} {{ node.revision.describe }}"
 
 device_type: qemu
 
-- 
2.39.2

