From 6fe30d4b28ef56659d2ce1eba6ae4f6a8c501c64 Mon Sep 17 00:00:00 2001
From: "kernelci.org bot" <bot@kernelci.org>
Date: Mon, 28 Feb 2022 11:52:42 +0000
Subject: [PATCH 2/3] STAGING config/kernelci.conf: use staging host and ports

---
 config/kernelci.conf | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/config/kernelci.conf b/config/kernelci.conf
index 287522aa1df7..21baff053b08 100644
--- a/config/kernelci.conf
+++ b/config/kernelci.conf
@@ -1,6 +1,6 @@
 [DEFAULT]
-db_config: docker-host
-storage_url: http://172.17.0.1:8002/
+db_config: staging.kernelci.org
+storage_url: http://staging.kernelci.org:9080
 verbose: true
 
 [trigger]
@@ -11,9 +11,9 @@ poll_period: 0
 kdir: /home/kernelci/data/linux
 output: /home/kernelci/data/output
 ssh_key: /home/kernelci/data/ssh/id_rsa_tarball
-ssh_port: 8022
+ssh_port: 9022
 ssh_user: kernelci
-ssh_host: 172.17.0.1
+ssh_host: staging.kernelci.org
 
 [runner]
 output: /home/kernelci/output
-- 
2.30.2

