From 305f82d4adbcaa3e6a372a3b154c7833f3cd2886 Mon Sep 17 00:00:00 2001
From: "kernelci.org bot" <bot@kernelci.org>
Date: Mon, 28 Feb 2022 11:58:45 +0000
Subject: [PATCH 3/3] STAGING config/kernelci.conf: monitor staging-mainline

---
 config/kernelci.conf | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/config/kernelci.conf b/config/kernelci.conf
index 3a57fc54cbf9..bc4e92d1ea1b 100644
--- a/config/kernelci.conf
+++ b/config/kernelci.conf
@@ -4,7 +4,7 @@ storage_url: http://staging.kernelci.org:9080
 verbose: true
 
 [trigger]
-build_config: mainline
+build_config: kernelci_staging-mainline
 poll_period: 0
 
 [tarball]
-- 
2.30.2

