From ee232ab1b58c9d1126b3ba24913ab70bcc45472f Mon Sep 17 00:00:00 2001
From: "kernelci.org bot" <bot@kernelci.org>
Date: Wed, 25 May 2022 09:17:18 +0000
Subject: [PATCH 4/4] STAGING config/kernelci.conf: add SMTP config to
 test_report

---
 config/kernelci.conf | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/config/kernelci.conf b/config/kernelci.conf
index e8e6047..dced512 100644
--- a/config/kernelci.conf
+++ b/config/kernelci.conf
@@ -26,8 +26,8 @@ lab_config: shell
 [kcidb]
 
 [test_report]
-email_host: your.host
-email_port: your.port
+email_host: smtp.gmail.com
+email_port: 465
 
 [db:docker-host]
 api: http://172.17.0.1:8001
-- 
2.30.2

