From 089877dd0d62b116bcfc7071a6dab3d9c4af70e2 Mon Sep 17 00:00:00 2001
From: Guillaume Tucker <guillaume.tucker@collabora.com>
Date: Fri, 3 Jan 2020 15:56:19 +0000
Subject: [PATCH 2/8] STAGING bisect.jpl: don't send emails to automatic
 recipients

---
 jenkins/bisect.jpl | 1 +
 1 file changed, 1 insertion(+)

diff --git a/jenkins/bisect.jpl b/jenkins/bisect.jpl
index c80cefc1c95f..57f1a81f1097 100644
--- a/jenkins/bisect.jpl
+++ b/jenkins/bisect.jpl
@@ -531,6 +531,7 @@ PYTHON_EGG_CACHE=${egg_cache} \
 --kdir=${kdir} \
 --subject=\"${subject}\" \
 --to=\"${params.EMAIL_RECIPIENTS}\" \
+--no-auto-recipients \
 """)
         }
     }
-- 
2.20.1

