From 7e0e9bb524378f9b42a28b744e845b32f8f8b036 Mon Sep 17 00:00:00 2001
From: "kernelci.org bot" <bot@kernelci.org>
Date: Mon, 2 Sep 2019 08:57:24 +0100
Subject: [PATCH 4/5] STAGING kernelci.config.test: debug noise in filters

---
 kernelci/config/test.py | 12 ++++++++++++
 1 file changed, 12 insertions(+)

diff --git a/kernelci/config/test.py b/kernelci/config/test.py
index 451e9606d4a6..ac544f8d13c9 100644
--- a/kernelci/config/test.py
+++ b/kernelci/config/test.py
@@ -401,6 +401,18 @@ class TestConfig(YAMLObject):
         return self._test_plans
 
     def match(self, arch, flags, config, plan=None):
+        if True:
+            print("{} {} (listed: {}), {} {}, {} {}, filters: {}".format(
+                plan,
+                (plan in self._test_plans and
+                 plan and self._test_plans[plan].match(config)),
+                plan in self._test_plans,
+                arch,
+                self.device_type.arch == arch,
+                config,
+                self.device_type.match(flags, config),
+                all(f.match(**config) for f in self._filters)
+            ))
         return (
             (plan is None or (
                 plan in self._test_plans and
-- 
2.20.1

