From 1b035c4ae7ab58398338066c15f4442ea5157894 Mon Sep 17 00:00:00 2001
From: "kernelci.org bot" <bot@kernelci.org>
Date: Tue, 15 Sep 2020 10:22:30 +0100
Subject: [PATCH 6/6] STAGING use staging.kernelci.org test-definitions

---
 templates/kselftest/kselftest.jinja2 | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/templates/kselftest/kselftest.jinja2 b/templates/kselftest/kselftest.jinja2
index c3ae8d75eddf..6812cac88eaa 100644
--- a/templates/kselftest/kselftest.jinja2
+++ b/templates/kselftest/kselftest.jinja2
@@ -17,7 +17,7 @@
 
     - repository: https://github.com/kernelci/test-definitions.git
       from: git
-      revision: kernelci.org
+      revision: staging.kernelci.org
       path: automated/linux/kselftest/kselftest.yaml
       name: kselftest
       parameters:
-- 
2.20.1

