From 9c319d50643c66a4c13525a131d8b74423f6258f Mon Sep 17 00:00:00 2001
From: "kernelci.org bot" <bot@kernelci.org>
Date: Thu, 30 Jan 2020 14:33:44 +0000
Subject: [PATCH] STAGING only send emails for kernelci tree

---
 jenkins/kernel-arch-complete.sh | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/jenkins/kernel-arch-complete.sh b/jenkins/kernel-arch-complete.sh
index 1d85ba267655..c53971109a4d 100755
--- a/jenkins/kernel-arch-complete.sh
+++ b/jenkins/kernel-arch-complete.sh
@@ -34,6 +34,12 @@ fi
 
     echo "Build has now finished, reporting result to dashboard."
     curl -X POST -H "Authorization: $EMAIL_AUTH_TOKEN" -H "Content-Type: application/json" -d '{"job": "'$TREE_NAME'", "kernel": "'$GIT_DESCRIBE'", "git_branch": "'$BRANCH'"}' ${API}/job
+
+if [ "$TREE_NAME" != "kernelci" ]; then
+    echo "Only dealing with kernelci tree on staging.kernelci.org"
+    exit 0
+fi
+
     if [ "$EMAIL" != "true" ]; then
         echo "Not sending emails because EMAIL was false"
         exit 0
-- 
2.20.1

