From 9f71e9ada752c9d5dfd11962d5dfe01d65a0f88e Mon Sep 17 00:00:00 2001
From: "kernelci.org bot" <bot@kernelci.org>
Date: Thu, 30 Jan 2020 14:33:44 +0000
Subject: [PATCH 6/9] STAGING only send emails for kernelci tree

---
 jenkins/kernel-arch-complete.sh | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/jenkins/kernel-arch-complete.sh b/jenkins/kernel-arch-complete.sh
index b6e61150d387..e9a720336eea 100755
--- a/jenkins/kernel-arch-complete.sh
+++ b/jenkins/kernel-arch-complete.sh
@@ -35,6 +35,11 @@ fi
     echo "Build has now finished, reporting result to dashboard."
     curl -X POST -H "Authorization: $EMAIL_AUTH_TOKEN" -H "Content-Type: application/json" -d '{"job": "'$TREE_NAME'", "kernel": "'$GIT_DESCRIBE'", "git_branch": "'$BRANCH'"}' ${API}/job
 
+if [ "$TREE_NAME" != "kernelci" ]; then
+    echo "Only dealing with kernelci tree on staging.kernelci.org"
+    exit 0
+fi
+
     if [ "$EMAIL" != "true" ]; then
         echo "Not sending emails because EMAIL was false"
         exit 0
-- 
2.20.1

