From 36e96ea8fd23163f1c999cae069d1ed764853538 Mon Sep 17 00:00:00 2001
From: Guillaume Tucker <guillaume.tucker@collabora.com>
Date: Tue, 22 Jan 2019 15:44:23 +0000
Subject: [PATCH 6/6] STAGING build.py: put compiler version in build log

---
 build.py | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/build.py b/build.py
index aed367424c61..ef54d96dd5cd 100755
--- a/build.py
+++ b/build.py
@@ -269,6 +269,9 @@ if not os.path.exists(kbuild_output):
 build_log = os.path.join(kbuild_output, "build.log")
 build_log_f = open(build_log, 'w', 0)
 
+# Put the compiler in the build log
+build_log_f.write("#\n# " + cc_version_full + "\n#\n")
+
 # ccache
 ccache = None
 ccache_dir = None
-- 
2.20.1

