From 7fc4647c3748e8a0af8e36158fa7076cfffbe32c Mon Sep 17 00:00:00 2001
From: "kernelci.org bot" <bot@kernelci.org>
Date: Thu, 30 Jan 2020 14:33:44 +0000
Subject: [PATCH 7/7] STAGING only send emails for kernelci tree

---
 jenkins/kernel-arch-complete.sh | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/jenkins/kernel-arch-complete.sh b/jenkins/kernel-arch-complete.sh
index 8e9dd6b8afbf..d46ac48decac 100755
--- a/jenkins/kernel-arch-complete.sh
+++ b/jenkins/kernel-arch-complete.sh
@@ -34,6 +34,12 @@ fi
 
     echo "Build has now finished, reporting result to dashboard."
     curl -X POST -H "Authorization: $EMAIL_AUTH_TOKEN" -H "Content-Type: application/json" -d '{"job": "'$TREE_NAME'", "kernel": "'$GIT_DESCRIBE'", "git_branch": "'$BRANCH'"}' ${API}/job
+
+if [ "$TREE_NAME" != "kernelci" ]; then
+    echo "Only dealing with kernelci tree on staging.kernelci.org"
+    exit 0
+fi
+
     if [ "$EMAIL" != "true" ]; then
         echo "Not sending emails because EMAIL was false"
         exit 0
-- 
2.20.1

