From baf18182c733e3357b1243300ed590131bb80756 Mon Sep 17 00:00:00 2001
From: Guillaume Tucker <guillaume.tucker@collabora.com>
Date: Tue, 22 Jan 2019 12:55:20 +0000
Subject: [PATCH 7/8] STAGING build.py -s to not build silently

---
 jenkins/build.jpl | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/jenkins/build.jpl b/jenkins/build.jpl
index c6c7c7a57fc6..35df22385a90 100644
--- a/jenkins/build.jpl
+++ b/jenkins/build.jpl
@@ -73,6 +73,8 @@ def buildConfig(config, kdir, kci_core) {
                        returnStdout: true)
     def opt = "-i -E ${params.CC}-${params.CC_VERSION}"
 
+    opt += " -s"  /* not silent */
+
     if (params.PUBLISH) {
         opt += " -e"
     } else {
-- 
2.20.1

