From 7cc415828571a9154c03da1f09aab287f663de61 Mon Sep 17 00:00:00 2001
From: "kernelci.org bot" <bot@kernelci.org>
Date: Tue, 7 Jul 2020 21:56:00 +0100
Subject: [PATCH 1/2] CHROMEOS bisect.jpl: add -chromeos in tags and email

---
 jobs/bisect.jpl | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/jobs/bisect.jpl b/jobs/bisect.jpl
index 7c5b4772aa3d..fff764260a2a 100644
--- a/jobs/bisect.jpl
+++ b/jobs/bisect.jpl
@@ -123,7 +123,7 @@ def createTag(kdir, iteration) {
     def tag = gitDescribe(kdir)
 
     dir(kdir) {
-        tag += "-${env.JOB_NAME}-${currentBuild.number}-${iteration}"
+        tag += "-${env.JOB_NAME}-chromeos-${currentBuild.number}-${iteration}"
         sh(script: "git tag -a ${tag} -m ${tag} HEAD")
     }
 
@@ -515,7 +515,7 @@ def bisectNext(kdir, status) {
 def pushResults(kci_core, kdir, checks, params_summary) {
     def subject = "\
 ${params.KERNEL_TREE}/${params.KERNEL_BRANCH} bisection: \
-${params.TEST_CASE} on ${params.TARGET}"
+${params.TEST_CASE} on ${params.TARGET} #${env.BUILD_NUMBER}-chromeos"
 
     dir(kci_core) {
         withCredentials([string(credentialsId: params.KCI_API_TOKEN_ID,
@@ -761,7 +761,7 @@ ${params_summary}""")
             currentBuild.result = "FAILURE"
 
             def tree_branch = "${params.KERNEL_TREE}/${params.KERNEL_BRANCH}"
-            def subject = "bisection error: #${env.BUILD_NUMBER} \
+            def subject = "bisection error: #${env.BUILD_NUMBER}-chromeos \
 ${tree_branch} ${params.LAB} ${params.TARGET}"
             def body = """\
 ${env.BUILD_URL}
-- 
2.20.1

