From 2cff8171efccd696371c7de2aaa2d706321b9879 Mon Sep 17 00:00:00 2001
From: "kernelci.org bot" <bot@kernelci.org>
Date: Tue, 7 Jul 2020 21:56:00 +0100
Subject: [PATCH 3/4] STAGING bisect.jpl: add -staging in tags and email

---
 jobs/bisect.jpl | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/jobs/bisect.jpl b/jobs/bisect.jpl
index c4a8f0d0f7be..4f62b67e8c57 100644
--- a/jobs/bisect.jpl
+++ b/jobs/bisect.jpl
@@ -125,7 +125,7 @@ def createTag(kdir, iteration) {
     def tag = gitDescribe(kdir)
 
     dir(kdir) {
-        tag += "-${env.JOB_NAME}-${currentBuild.number}-${iteration}"
+        tag += "-${env.JOB_NAME}-staging-${currentBuild.number}-${iteration}"
         sh(script: "git tag -a ${tag} -m ${tag} HEAD")
     }
 
@@ -553,7 +553,7 @@ def bisectNext(kdir, status) {
 def pushResults(kci_core, kdir, checks, params_summary) {
     def subject = "\
 ${params.KERNEL_TREE}/${params.KERNEL_BRANCH} bisection: \
-${params.TEST_CASE} on ${params.TARGET}"
+${params.TEST_CASE} on ${params.TARGET} #${env.BUILD_NUMBER}-staging"
 
     dir(kci_core) {
         withCredentials([string(credentialsId: params.KCI_API_TOKEN_ID,
@@ -799,7 +799,7 @@ ${params_summary}""")
             currentBuild.result = "FAILURE"
 
             def tree_branch = "${params.KERNEL_TREE}/${params.KERNEL_BRANCH}"
-            def subject = "bisection error: #${env.BUILD_NUMBER} \
+            def subject = "bisection error: #${env.BUILD_NUMBER}-staging \
 ${tree_branch} ${params.LAB} ${params.TARGET}"
             def body = """\
 ${env.BUILD_URL}
-- 
2.30.2

