From b77111d9a95a0aab24809ced7a38e9ff5cb16b26 Mon Sep 17 00:00:00 2001
From: "kernelci.org bot" <bot@kernelci.org>
Date: Fri, 11 Feb 2022 20:20:01 +0000
Subject: [PATCH 4/4] STAGING rootfs-builder.jpl: use staging branch

---
 jobs/rootfs-builder.jpl | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/jobs/rootfs-builder.jpl b/jobs/rootfs-builder.jpl
index e4ba8eaf4bfc..894efe7236b9 100644
--- a/jobs/rootfs-builder.jpl
+++ b/jobs/rootfs-builder.jpl
@@ -47,7 +47,7 @@ import org.kernelci.util.Job
 def build_buildroot(config, arch, pipeline_version, kci_core) {
     dir(kci_core) {
         sh(script: """\
-git clone https://github.com/kernelci/buildroot
+git clone https://github.com/kernelci/buildroot -b staging.kernelci.org
 ./kci_rootfs \
 build \
 --rootfs-config ${config} \
-- 
2.30.2

