From b8985317a930ea3551c0a49ddf3d9fc6526fe2e7 Mon Sep 17 00:00:00 2001
From: Guillaume Tucker <guillaume.tucker@collabora.com>
Date: Fri, 16 Nov 2018 11:12:34 +0000
Subject: [PATCH] HACK allow sending the same report multiple times

---
 app/handlers/send.py | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/app/handlers/send.py b/app/handlers/send.py
index 87b5d1db5ce7..bf90789d6676 100644
--- a/app/handlers/send.py
+++ b/app/handlers/send.py
@@ -188,7 +188,10 @@ class SendHandler(hbase.BaseHandler):
                     'email', report_type, job, branch, kernel])
 
                 with redis.lock.Lock(self.redisdb, lock_key, timeout=2):
-                    if not self.redisdb.exists(schedule_hash):
+                    #if not self.redisdb.exists(schedule_hash):
+                    if self.redisdb.exists(schedule_hash):
+                        self.log.warn("deBug already done that one...")
+                    if True:
                         self.redisdb.set(
                             schedule_hash, "schedule", ex=86400)
 
-- 
2.20.1

